doc/api/esm.md

```diff
@@ -313,50 +275,337 @@ If a package has no exports, setting `"exports": false` can be used instead of
 `"exports": {}` to indicate the package does not intend for submodules to be
 exposed.
 
-Exports can also be used to map the main entry point of a package:
+Any invalid exports entries will be ignored. This includes exports not
+starting with `"./"` or a missing trailing `"/"` for directory exports.
+
+Array fallback support is provided for exports, similarly to import maps
```

Comment:
confirming: is this still true for import maps?

Reply:
Fallbacks are still considered a possible future feature of import maps, but yes perhaps we should reword this.